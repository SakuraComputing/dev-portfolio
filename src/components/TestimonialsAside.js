import React from 'react';

export const TestimonialsAside = () => (
    <aside className="testimonials aside section">
        <div className="section-inner">
            <h2 className="heading">Testimonials</h2>
            <div className="content">
                <div className="item">
                    <blockquote className="quote">
                        <p><i className="fa fa-quote-left"></i>James is an excellent software engineer and he is
                            passionate about what he does. You can totally count on him to deliver your projects!</p>
                    </blockquote>
                    <p className="source"><span className="name">Tim Adams</span><br/><span className="title">Curabitur commodo</span>
                    </p>
                </div>

                <p><a className="more-link" href="#"><i className="fa fa-external-link"></i> More on Linkedin</a></p>

            </div>
        </div>
    </aside>

);

export default TestimonialsAside;