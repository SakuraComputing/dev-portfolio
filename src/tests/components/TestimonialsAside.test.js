import React from 'react';
import { shallow } from 'enzyme';
import TestimonialsAside from '../../components/TestimonialsAside';

describe('TestimonialsAside Section', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<TestimonialsAside />);
        expect(wrapper).toMatchSnapshot();
    });
});