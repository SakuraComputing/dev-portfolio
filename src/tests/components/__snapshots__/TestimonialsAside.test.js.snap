// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestimonialsAside Section should render correctly 1`] = `
<aside
  className="testimonials aside section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      Testimonials
    </h2>
    <div
      className="content"
    >
      <div
        className="item"
      >
        <blockquote
          className="quote"
        >
          <p>
            <i
              className="fa fa-quote-left"
            />
            James is an excellent software engineer and he is passionate about what he does. You can totally count on him to deliver your projects!
          </p>
        </blockquote>
        <p
          className="source"
        >
          <span
            className="name"
          >
            Tim Adams
          </span>
          <br />
          <span
            className="title"
          >
            Curabitur commodo
          </span>
        </p>
      </div>
      <p>
        <a
          className="more-link"
          href="#"
        >
          <i
            className="fa fa-external-link"
          />
           More on Linkedin
        </a>
      </p>
    </div>
  </div>
</aside>
`;
